src/any_agent/config.py

```diff
@@ -27,6 +27,7 @@ class MCPSseParams(BaseModel):
 
 
 class TracingConfig(BaseModel):
+    output_dir: str = "traces"
```

Comment:
@njbrake WDYT if we allow this to be `None` as a way to disable the JSON exporter?
Feel like it might be useful for someone to have a straightforward way to disable it and would be consistent with the other props (None means disabled)

Reply:
Sounds good to me 👍